### DISCUSSION TITLE
feature: add opencontainers label to kong docker image

### DISCUSSION THREAD

**Discussion by mikaello** [Kong/kong#9982] [Ideas and feature requests]
I would like the distributed docker image to contain the `org.opencontainers.image.source` label for easy reference back to this repository. This is useful for tools like Renovate that makes updates to repositories when this image gets updates. OCI has standardized a few labels for adding metadata to images: https://github.com/opencontainers/image-spec/blob/main/annotations.md

Currently this is the list of labels:

```shell
$ docker inspect kong:latest --format '{{ json .Config.Labels }}' | jq
```

```json
{
  "maintainer": "Kong Docker Maintainers <docker@konghq.com> (@team-gateway-bot)"
}
```

**Comment by mikaello** [comment]
Instead of manually adding the label, you could also use the Docker metadata Github action: https://github.com/docker/metadata-action

**Comment by ghost** [comment]
What labels are usually added?

**Reply by mikaello** [reply]
Hm, I haven't seen a common pattern, but the one proposed here is quite common (`.source`), the docker metadata action would add these:

-  org.opencontainers.image.created
-  org.opencontainers.image.description
-  org.opencontainers.image.licenses
-  org.opencontainers.image.revision
-  org.opencontainers.image.source,
-  org.opencontainers.image.title
-  org.opencontainers.image.url
-  org.opencontainers.image.version

**Reply by mikaello** [reply]
I notice that a couple of labels are added now:

```shell
$ docker inspect kong:latest --format '{{ json .Config.Labels }}'
{
  "maintainer": "Kong Docker Maintainers <docker@konghq.com> (@team-gateway-bot)",
  "org.opencontainers.image.ref.name": "ubuntu",
  "org.opencontainers.image.version": "22.04"
}
```

But it would be really nice if also the `org.opencontainers.image.source` label could be added.

**Reply by mikaello** [reply]
Any chance `org.opencontainers.image.source` could be added?
